lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,8 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+    $this->northstar_id = $data->field_northstar_id_value;
```

Comment:
for context, this comment was made because before the northstar_id was only added with the competition parameter, but we decided to make it always added (and thus this isnt a problem anymore)